### ISSUE TITLE
Flexible room size

### ISSUE THREAD

**Issue by leny** [GeoGuess/Geoguess#89]
Hi, thanks for this great project!

I will try to see if I can tweak that in the code, but maybe you can tell me if it's possible to implement that feature: at the moment, we can only create a room for a specific amount of player.  
It could be great to have more feedback on the "waiting modal", to see how many (and maybe the nicknames) of the players already connected, with a button to start the game at any point.

Thanks in advance, I will make a pull request if I can. :)

**Comment by BilelJegham** [comment]
Hi, great idea.

The roomSize is used to end the room.
But I think you can add a step in the room definition. `src/components/widgets/dialog/DialogRoom.vue`

**Comment by wlanowski** [comment]
If Flexible room size is working, idea:
Creating loby with running rooms with possibility to join and to kick players. Optional password protecting of rooms.

**Comment by supersational** [comment]
Just a :+1: to this, I almost gave up trying to get 8 people to join a game, managed after a few attempts and the experience was then flawless so thanks to the devs!

The issue was it wasn't easy to see who we were waiting for, so a list of names of players who've joined would be super helpful.. In the confusion I think some people may have joined twice which then prevented others from joining.

Another reason this would be helpful is that a lot of people starting/quitting games can quickly burn through your Maps API free quota, since I think each start of a new game instantiates a new map?

**Comment by leny** [comment]
Hi, I finally _kinda_ gave up on this... I started digging into the codebase, but I never really work with VueJS before and I don't have the time right now to do the switch.  
Moreover, one of my student (I also teach web dev in a bootcamp-style school) suggest the idea to make our own clone of Geoguessr as a pedagogic project.

So, yes, I fixed the issue... on ["my" version of the game](//github.com/leny/enigjewo), this time in react.

Regarding the flexible room size, the idea is really easy: creator of the room don't fix a size, see the name of players joining in a "lobby" screen, and then can start the game at any moment.
